nucliadb_utils/src/nucliadb_utils/audit/stream.py

```diff
@@ -58,6 +58,7 @@ class RequestContext:
     def __init__(self):
         self.audit_request: AuditRequest = AuditRequest()
         self.start_time: float = time.monotonic()
+        self.path: str = ""
```

Comment:
Why do you need `self.path` for? I'm not finding where is this used

Reply:
It is used below
Here:
`context.path = request.url.path`

And here:
```
if "/ask" in context.path:
    auditrequest.type = AuditRequest.CHAT
else:
    auditrequest.type = AuditRequest.SEARCH
```